test/replcompletions.jl

```diff
@@ -744,6 +744,13 @@ function test_dict_completion(dict_name)
     c, r = test_complete(s)
     @test c == Any[":α]"]
 end
+
+if Base.have_color # The test is only meaningful when Base.have_color==true
```

Comment:
Yes it would pass the test when `Base.have_color=false`. I just found it more meaningful to test if there is a possibility of failure.